src/measure_theory/function/l1_space.lean

```diff
@@ -560,6 +579,10 @@ begin
     (integrable_zero _ _ _) hf,
 end
 
+lemma integrable_finset_sum' {ι} (s : finset ι)
+  {f : ι → α → β} (hf : ∀ i ∈ s, integrable (f i) μ) : integrable (∑ i in s, f i) μ :=
+by { convert integrable_finset_sum s hf, ext x, simp }
```

Comment:
I think it would be shorter to swap these two lemmas around; as far as I can tell, this lemma is the goal state at the `exact` above.